jobs/rocoto/ocnpost.sh

```diff
@@ -89,13 +89,10 @@ else
     export COMIN="$ROTDIR/$CDUMP.$PDY/$cyc"
     export COMOUT="$ROTDIR/$CDUMP.$PDY/$cyc"
 fi
-[[ ! -d $COMOUT ]] && mkdir -m 775 -p $COMOUT
+[[ ! -d $COMOUT/ocean ]] && mkdir -m 775 -p $COMOUT/ocean
+[[ ! -d $COMOUT/ice ]] && mkdir -m 775 -p $COMOUT/ice
```

Comment:
why is the `-m 755` necessary?

Reply:
I suggest removing it.

Reply:
Going to wait for @JessicaMeixner-NOAA to chime in just in case the more open permissions are needed for the prototype work. Even if it is needed, this is probably not the best ones to use. For reference, atmos has the following permissions: `rwxr-s---`, compared to the `rwxrwsr-x` for the other directories (wave also uses these broader permissions).

Reply:
I don't know of an issue/reason for needing this. Thanks for checking w/me.

Reply:
Done, though FWIW the wave mkdirs also set the permissions, so we'll want to revisit that later.